lib/praxis/restful_doc_generator.rb

```diff
@@ -1,4 +1,26 @@
 module Praxis
```

Comment:
ping! let's remove any changes to this file. I'm pretty sure they're a mistake/not-needed